var/spack/repos/builtin/packages/hiredis/package.py

```diff
@@ -6,21 +6,16 @@
 from spack.package import *
 
 
-class Hiredis(CMakePackage, MakefilePackage):
+class Hiredis(MakefilePackage, CMakePackage):
     """Hiredis is a minimalistic C client library for the Redis database."""
 
     homepage = "https://github.com/redis/hiredis"
-    url = "https://github.com/redis/hiredis/archive/v0.14.1.tar.gz"
-    git = "https://github.com/redis/hiredis"
+    url = "https://github.com/redis/hiredis/archive/refs/tags/v1.0.2.tar.gz"
+    git = "https://github.com/redis/hiredis.git"
 
-    maintainers = ["rblake-llnl"]
-
-    build_system(
-        conditional("cmake", when="@1:"),
-        conditional("makefile", when="@0"),
-        default="cmake",
-    )
+    maintainers = ["lpottier"]
```

Comment:
You can be sole maintainer, or you can add me to help share the load.  I think you're more familiar with the package.

Reply:
I am good with sharing the load! I will add you.